tutorial.ipynb

```diff
@@ -0,0 +1,665 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Spectral Event Analysis Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial is a hands-on introduction to using the [SpectralEvents toolbox](https://github.com/jonescompneurolab/SpectralEvents), **a collection of functions designed to help researchers characterize high-amplitude peaks in the spectral representation of continuous neural signals**. Here, we'll load and analyze data that is distributed with the toolbox, as described in [Shin et al. (2017)](https://doi.org/10.7554/eLife.29086)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we'll import some dependencies. Note that `seaborn` is not required, but is added to improve plot styling."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "import sys\n",
+    "import os.path as op\n",
+    "from glob import glob\n",
+    "\n",
+    "import numpy as np\n",
+    "from scipy.io import loadmat\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
```

Comment:
seaborn is not in `requirements.txt`

Maybe comment it out if you don't want to make it a hard dependency?